rest_framework/fields.py

```diff
@@ -1105,7 +1106,10 @@ def enforce_timezone(self, value):
         field_timezone = getattr(self, 'timezone', self.default_timezone())
 
         if (field_timezone is not None) and not timezone.is_aware(value):
-            return timezone.make_aware(value, field_timezone)
+            try:
+                return timezone.make_aware(value, field_timezone)
+            except Exception:
```

Comment:
What sort of exception would there be without pytz ?
If the issue is just the non requirement of pytz, we have a compat module that will deal with that.